### PR TITLE
Update electron to 1.2.2

### PR DESCRIPTION
**Pull request by victorporof**
Signed-off-by: Victor Porof <vporof@mozilla.com>

@Mossop wanna do your magic?

Error: GET https://mozilla-releng-tofino.s3.amazonaws.com/mozilla/tofino-electron/ef9515aac9ea/electron-v1.2.2-darwin-x64.zip returned 403

### CODE REVIEW

**Comment by victorporof** [comment]
Fixes #651 which blocks #609

**Comment by Mossop** [comment]
I have linux and 64-bit windows builds done. @bgrins, can you do an OSX build off the latest tofino branch of tofino-electron and mail it to me?

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "repository": "https://github.com/mozilla/tofino",
   "_electron": {
     "mirror": "https://mozilla-releng-tofino.s3.amazonaws.com/mozilla/tofino-electron/",
-    "version": "1.2.1",
+    "version": "1.2.2",
     "revision": "ef9515aac9ea"
```
Comment (Mossop) [inline package.json]: Need to update this to 5cd1dc12f984

**Comment by victorporof** [comment]
Done!

**Pull request merged**